api/comments.rb

```diff
@@ -62,5 +62,4 @@
       pass
     end
   end
-  comment.to_hash.to_json
```

Comment:
@dianakhuang: We want to ensure that edx-platform really needs this to not get an error on delete.  It might be a simple fix on edx-platform side.

Otherwise, we need something like:
parent_id = comment.parent_id		   
comment_as_json = comment.to_hash.to_json
comment.destroy
...
comment_as_json
```